### PR TITLE
e2e: Fix operator pod prefix

### PR DESCRIPTION
**Pull request by mrogers950**
@sallyom

### CODE REVIEW

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 const (
 	serviceCAOperatorNamespace   = "openshift-service-ca-operator"
 	serviceCAControllerNamespace = "openshift-service-cert-signer"
-	serviceCAOperatorPodPrefix   = "service-ca-operator"
+	serviceCAOperatorPodPrefix   = "openshift-service-ca-operator"
```
Comment (ericavonb) [inline test/e2e/e2e_test.go]: we should import these from the `operatorclient` package: https://github.com/openshift/service-ca-operator/blob/a92546fbaa0c89374c4b4981f57e5b44aae88590/pkg/operator/operatorclient/interfaces.go#L6

**Comment by mrogers950** [comment]
@ericavonb updated

File: test/e2e/e2e_test.go [hunk 2]
```diff
@@ -7,20 +7,21 @@ import (
...
 const (
-	serviceCAOperatorNamespace   = "openshift-service-ca-operator"
-	serviceCAControllerNamespace = "openshift-service-cert-signer"
-	serviceCAOperatorPodPrefix   = "service-ca-operator"
+	serviceCAOperatorNamespace   = operatorclient.OperatorNamespace
+	serviceCAOperatorPodPrefix   = operatorclient.OperatorNamespace // Same as operator namespace
+	serviceCAControllerNamespace = operatorclient.TargetNamespace
 	apiInjectorPodPrefix         = "apiservice-cabundle-injector"
 	configMapInjectorPodPrefix   = "configmap-cabundle-injector"
 	caControllerPodPrefix        = "service-serving-cert-signer"
```
Comment (ericavonb) [inline test/e2e/e2e_test.go]: Optional: we can use these too. https://github.com/openshift/service-ca-operator/blob/a92546fbaa0c89374c4b4981f57e5b44aae88590/pkg/controller/api/resourcenames.go#L22-L25

Comment (mrogers950) [reply test/e2e/e2e_test.go]: done

**Pull request merged**